listen.py

```diff
@@ -43,6 +43,41 @@ def from_json(cls, j):
             data=j.get('track_metadata')
         )
 
+    @classmethod
+    def from_influx(cls, row):
+        """ Factory to make Listen objects from an influx row
+        """
+        dt = datetime.strptime(row['time'] , "%Y-%m-%dT%H:%M:%SZ")
+        t = int(dt.strftime('%s'))
+        mbids = []
+        if row.get('artist_mbids'):
+            for mbid in row.get('artist_mbids').split(","):
+                mbids.append(mbid) 
+        tags = []
+        if row.get('tags'):
+            for tag in row.get('tags').split(","):
+                tags.append(tag)
+
+        data = {
+            'artist_mbids' : mbids,
+            'album_msid' : row.get('album_msid'),
+            'album_mbid' : row.get('album_mbid'),
+            'album_name' : row.get('album_name'),
+            'recording_mbid' : row.get('recording_mbid'),
+            'tags' : tags,
+        }
+        return cls(
+            timestamp = t,
```

Comment:
small formatting thing, these are arguments to a function, and so the standard is to have no spaces - 
```python
cls(timestamp=t, x=y, ...)
```